src/Providers/HydratorProvider.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Choredo\Providers;
+
+use Choredo\Hydrators\FamilyHydrator;
+use League\Container\ServiceProvider\AbstractServiceProvider;
+
+class HydratorProvider extends AbstractServiceProvider
```

Comment:
This class isn't necessary now that you're sharing it from the middleware provider, no?

Reply:
:+1: